### ISSUE TITLE
License for page

### ISSUE THREAD

**Issue by matthiasbeyer** [hfuunfug/unfug.org#13]
What license shall we put on the page?

**Comment by fkemmer** [comment]
Do we really need a licence? The rights to the talk should remain with the authors, so the only thing really left that's "owned" by us would be the blog (and, depending on how much you modified, the code). I'd suggest some Creative Common. Personally, I'm a fan of [CC-BY-SA](https://creativecommons.org/licenses/by-sa/4.0/).

**Comment by matthiasbeyer** [comment]
I'd agree, but I'd also add a "NC" part to the license.

**Comment by fkemmer** [comment]
according to CC it's not a "Free Culture License" anymore when the NC part is added.

**Comment by matthiasbeyer** [comment]
Why is that?

I would argue that we are a group which shares knowledge for free (as in free beer), so we should share the notes and schedules for our talks under the same terms and ensure they are shared in the same spirit - free as in free beer.

**Comment by fkemmer** [comment]
well... don't ask me. I think the "same spirit" is covered by SA: So even if you incorporate our "knowledge" into commercial products, you'd still have to share the source under a CC-Licence. Isn't it the same with GPL?

**Comment by matthiasbeyer** [comment]
"NC" says that you are not allowed to use the knowlegde we publish on our page for commercial things. So you aren't even allowed to put things from us into a commerical product. I think this is even more restrictive than the GPL.

**Comment by fkemmer** [comment]
exactly. Which is why I would drop the NC and make it a "Free Culture License" ;)
